mlocal/frags/Makefile.stub

```diff
@@ -73,6 +76,18 @@ $(go_OBJ): $(BUILDDIR)/vendors-done $(libruntime) $(BUILDDIR)/config.h
 	@rm -f $(go_OBJ)
 	$(V)export BUILDDIR=`readlink -f $(BUILDDIR)` && cd $(SOURCEDIR)/src/pkg/buildcfg && go generate
 
+# bash_completion file
+$(bash_completion): $(go_OBJ)
+	@rm -f $(bash_completion)
+	@mkdir -p $(SOURCEDIR)/etc/bash_completion.d 
```

Comment:
If it's a generated file, it should end up in BUILDDIR not SOURCEDIR, that's why I commented earlier on the addition of source file to CLEANFILES.